CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## Unreleased
+- Added folder and folder id to api datasets files list [#34](https://github.com/clowder-framework/clowder/issues/34)
```

Comment:
can you update swagger and add the returned json document to the correct function.

Reply:
i'm seeing related methods starting on line

2643 
and
2530.

The response just says '200' and 'ok.' This is in the swagger.yml. Am I looking in the wrong place to change this?

Reply:
@tcnichol Rob is talking about the return type. We don't do this for most of the endpoints. I made a new task for doing that https://github.com/clowder-framework/clowder/issues/247. Since we want to merge this today I think we are good without it (I tried to do it but ran out of time and didn't want to accidentally break the docs in a hurry).